pyproject.toml

```diff
@@ -45,6 +45,7 @@ dynamic = ["version"]
 [project.optional-dependencies]
 test = [
     "pytest",
+    "essspectroscopy @ git+https://github.com/scipp/essspectroscopy.git@main",
```

Comment:
Not sure it should be added to the main deps. Only needed when running the Bifrost detector & reduction services.

Reply:
We should probably have instrument specific `optional-dependencies` extras.

Reply:
Added.